lib/lita/handlers/locker_misc.rb

```diff
@@ -58,6 +58,11 @@ def dequeue(response)
         return response.reply(t('subject.does_not_exist', name: name)) unless Label.exists?(name)
         l = Label.new(name)
         l.wait_queue.delete(response.user.id)
+        queued = l.wait_queue.to_a
+        l.wait_queue.clear
+        queued.chunk { |x| x }.map(&:first).each do |user|
+          l.wait_queue << user
+        end
```

Comment:
Seems OK, but if this logic gets much bigger, you may want to move this to a method on the Label object instead for a better separation of concerns.